### PR TITLE
feat : Shown version information when the program started

### PR DESCRIPTION
**Pull request by huahu0213**
Please answer these questions before submitting a pull request

- Why submit this pull request?
    - [x] New feature provided

- Related issues
   Version information should be shown when the program started #228

### New feature or improvement
    - When apisix ingress controller started, print the "version.Long()" information

### CODE REVIEW

File: cmd/ingress/ingress.go [hunk 1]
```diff
@@ -96,6 +97,8 @@ the apisix cluster and others are created`,
 			log.DefaultLogger = logger
 			log.Info("apisix ingress controller started")
 
+			log.Info("version: ", version.Short())
```
Comment (tokers) [inline cmd/ingress/ingress.go:100]: I think here output the long version is better :)

**Comment by gxthrj** [comment]
e2e test failed

File: cmd/ingress/ingress.go (hunk 1 above)
Comment (huahu0213) [reply cmd/ingress/ingress.go:100]: OK，I will deal with this

**Comment by tokers** [comment]
@huahu0213 Please merge the newest master branch.

**Review by tokers — APPROVED** [review]

**Comment by tokers** [comment]
@huahu0213 Merged and thanks for your good job!

**Pull request merged**